docs/topics/mta-web-reviewing-a-task-log.adoc

```diff
@@ -0,0 +1,17 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-01
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-web-reviewing-a-task-log_{context}"]
+= Reviewing a task log
+
+To find details and logs of a particular {ProductFullName} task, you can use the *Task Manager* page. 
+
+.Procedure
+
+. In the *Migration* view, click *Task Manager*.
+. Open the *Task Manager* page.
+. Click the *Options* menu ({kebab}).
```

Comment:
@mpershina please mention that this is for a task row
`Click the Options menu ( kebab ) for the selected task. ` (copied from next sub-section)